relay-server/src/metrics_extraction/transactions.rs

```diff
@@ -160,23 +611,33 @@ fn get_transaction_name(
     name
 }
 
+#[derive(Default, Clone)]
+struct UniversalTags {
+    release: Option<String>,
+    dist: Option<String>,
+    environment: Option<String>,
+    transaction: Option<String>,
+    platform: Option<String>,
+    transaction_status: Option<String>,
+    transaction_op: Option<String>,
+    http_method: Option<String>,
+    other: BTreeMap<String, String>,
+}
+
```

Comment:
nit: Let's put all these types into a separate module:

```
metrics_extraction/
    transactions/
        mod.rs
        types.rs
```

(same for sessions).